server/controller/getQuestions.js

```diff
@@ -0,0 +1,20 @@
+const getQuestionsQuery = require('../database/queries/getQuestionsQuery');
+
+const getQuestions = (req, res) => {
+	getQuestionsQuery()
+		.then((result) => {
+			if (!result) {
+				return res
```

Comment:
this is not  a server error. This is either a bad request or a no-content response. Look them up.